### PR TITLE
Update documentation with initial PDP [6457]

### PR DESCRIPTION
**Pull request by LuisGP**
This PR adds documentation of #724.

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/63/)

File: docs/advanced.rst [hunk 1]
```diff
@@ -586,6 +586,7 @@ The discovery mechanism is divided into two phases: Participant Discovery Phase
     Before discovering any entity of a remote participant, both participants have to meet between them.
     Participant Discovery Phase provides this step and is responsible for sending periodic information about itself.
     To know how to configure where to send this periodic information, see :ref:`initial-peers`.
+    To know how to configure these announcements period and count, see :ref:`Discovery Configuration <dconf>`.
```
Comment (raquelalvarezbanos) [inline docs/advanced.rst]:
```suggestion
    To know how to configure the announcement period and count, see :ref:`Discovery Configuration <dconf>`.
```

File: code/XMLTester.xml [hunk 2]
```diff
@@ -1008,14 +1008,18 @@
...
         </leaseDuration>
 
-        <leaseAnnouncement>  
+        <leaseAnnouncement>
             <!-- DURATION -->
         </leaseAnnouncement>
 
+        <initialAnnouncements>
+            <!-- DURATION -->
+        </initialAnnouncements>
```
Comment (raquelalvarezbanos) [inline code/XMLTester.xml]: When we say `<!-- DURATION -->` inside `<initialAnnouncements>`, is it correct? I thought we would have `<count>` and `<period>` instead

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/64/)

File: code/XMLTester.xml (hunk 2 above)
Comment (LuisGP) [reply code/XMLTester.xml]: Isn't correct. Nice catch!

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/65/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/66/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/76/)

**Comment by LuisGP** [comment]
Test passing in local.

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/99/)

File: docs/xmlprofiles.rst [hunk 3]
```diff
@@ -779,36 +779,40 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
...
+|                            |   StaticEDP based on an XML           |                         |                      |
+|                            |   file would be used with the         |                         |                      |
+|                            |   contents of                         |                         |                      |
+|                            |   ``<staticEndpointXMLFilename>``.    |                         |                      |
++----------------------------+---------------------------------------+-------------------------+----------------------+
+| ``<simpleEDP>``            | Attributes of the SimpleEDP           | :ref:`simpleEDP <sedp>` |                      |
+|                            | protocol                              |                         |                      |
++----------------------------+---------------------------------------+-------------------------+----------------------+
+| ``<leaseDuration>``        | Indicates how long this               |  :ref:`DurationType`    | 130 s                |
```
Comment (MiguelCompany) [inline docs/xmlprofiles.rst]: This default value has changed to 20 s.

File: docs/xmlprofiles.rst [hunk 4]
```diff
@@ -779,36 +779,40 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
...
+|                            | RTPSParticipant should consider       |                         |                      |
+|                            | remote RTPSParticipants alive.        |                         |                      |
++----------------------------+---------------------------------------+-------------------------+----------------------+
+| ``<leaseAnnouncement>``    | The period for the RTPSParticipant    |  :ref:`DurationType`    | 40 s                 |
```
Comment (MiguelCompany) [inline docs/xmlprofiles.rst]: This default value has changed to 3 s.

File: docs/xmlprofiles.rst [hunk 5]
```diff
@@ -824,6 +828,24 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
...
 +---------------------------+---------------------------------------------+-------------+---------------+
 
+.. _InitAnnounce:
+
+**Initial Announcements**
+
++---------------------------+---------------------------------------------+---------------------+---------------+
+| Name                      | Description                                 | Values              | Default       |
++===========================+=============================================+=====================+===============+
+| ``<count>``               | Number of Discovery Messages to send at the | ``Uint32``          | 0             |
```
Comment (MiguelCompany) [inline docs/xmlprofiles.rst]: This default value is 5.

File: docs/xmlprofiles.rst [hunk 6]
```diff
@@ -824,6 +828,24 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
...
+|                           | period specified by ``<period>``.           |                     |               |
+|                           | After these announcements, the              |                     |               |
+|                           | RTPSParticipant will continue sending its   |                     |               |
+|                           | Discovery Messages at the                   |                     |               |
+|                           | ``<leaseAnnouncement>`` rate.               |                     |               |
++---------------------------+---------------------------------------------+---------------------+---------------+
+| ``<period>``              | The period for the RTPSParticipant to send  | :ref:`DurationType` | 500 ms        |
```
Comment (MiguelCompany) [inline docs/xmlprofiles.rst]: The default value is 100 ms

File: docs/xmlprofiles.rst (hunk 3 above)
Comment (LuisGP) [inline docs/xmlprofiles.rst]:
```suggestion
| ``<leaseDuration>``        | Indicates how long this               |  :ref:`DurationType`    |  20 s                |
```

File: docs/xmlprofiles.rst (hunk 4 above)
Comment (LuisGP) [inline docs/xmlprofiles.rst]:
```suggestion
| ``<leaseAnnouncement>``    | The period for the RTPSParticipant    |  :ref:`DurationType`    |  3 s                 |
```

File: docs/xmlprofiles.rst (hunk 5 above)
Comment (LuisGP) [inline docs/xmlprofiles.rst]:
```suggestion
| ``<count>``               | Number of Discovery Messages to send at the | ``Uint32``          | 5             |
```

File: docs/xmlprofiles.rst (hunk 6 above)
Comment (LuisGP) [inline docs/xmlprofiles.rst]:
```suggestion
| ``<period>``              | The period for the RTPSParticipant to send  | :ref:`DurationType` | 100 ms        |
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/100/)

**Pull request merged**